### ISSUE TITLE
[BUG] Constructing a Python Column from a gdf_column redundantly recomputes the null count

### ISSUE THREAD

**Issue by jrhemstad** [rapidsai/cudf#2333]
**Describe the bug**
Constructing a Python `Column` object from a `gdf_column` C++ object looks like this:

```
cdef gdf_column* output;
... // output comes from libcudf
data, mask = gdf_column_to_column_mem(output)
return Column.from_mem_views(data, mask)
```

The problem with this is that it ignores the `null_count` in the `output` column and then redundantly recomputes it in Column `__init__`.

**Expected behavior**
Column `__init__` should optionally accept a null count that skips redundantly recomputing it from the bitmask.

**Comment by galipremsagar** [comment]
Fixing it in #2329

**Comment by kkraus14** [comment]
Fixed by #2329
